python_docs_theme/layout.html

```diff
@@ -38,11 +38,71 @@
 {% block extrahead %}
     <link rel="shortcut icon" type="image/png" href="{{ pathto('_static/' + theme_root_icon, 1) }}" />
     {% if builder != "htmlhelp" %}
-    {% if not embedded %}<script type="text/javascript" src="{{ pathto('_static/copybutton.js', 1) }}"></script>{% endif %}
+        {% if not embedded %}
+            <script type="text/javascript" src="{{ pathto('_static/copybutton.js', 1) }}"></script>
+            <script type="text/javascript" src="{{ pathto('_static/menu.js', 1) }}"></script>
+        {% endif %}
     {% endif %}
     {{ super() }}
 {% endblock %}
 
+{%- block css %}
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    {{ super() }}
+{%- endblock %}
+
+{%- block body_tag %}
+    <body>
+    <nav class="mobile-nav">
+        <input type="checkbox" class="toggler">
+        <div class="hamburger">
+            <div></div>
+        </div>
+        <div class="nav-content">
+            <img src="{{ pathto('_static/' + theme_root_icon, 1) }}" alt="Logo"/>
+            {%- if switchers is defined %}
+            <div class="version_switcher_placeholder">{{ release }}</div>
+            {% endif -%}
+            {%- if pagename != "search" and builder != "singlehtml" %}
+                <div id="searchbox" style="display: none" role="search">
+                    <form class="search" action="{{ pathto('search') }}" method="get">
+                        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"
+                             class="search-icon">
+                            <path fill-rule="nonzero"
+                                  d="M15.5 14h-.79l-.28-.27a6.5 6.5 0 001.48-5.34c-.47-2.78-2.79-5-5.59-5.34a6.505 6.505 0 00-7.27 7.27c.34 2.8 2.56 5.12 5.34 5.59a6.5 6.5 0 005.34-1.48l.27.28v.79l4.25 4.25c.41.41 1.08.41 1.49 0 .41-.41.41-1.08 0-1.49L15.5 14zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"></path>
+                        </svg>
+                        <label id="mobilesearchlabel" for="searchbox">{{ _('Quick search') }}</label>
+                        <input type="text" name="q" aria-labelledby="searchlabel"/>
+                        <input type="submit" value="{{ _('Go') }}"/>
+                    </form>
+                </div>
+                <script>document.getElementById('searchbox').style.display='block';</script>
```

Comment:
Any specific reason to use this at all as `#searchbox` is already a block or not at least put it in the stylesheet

Reply:
I was trying to remove unnecessary use of jquery, as I read that this is planned in the future. However, I can make a PR later specifically addressing jquery removal.

Reply:
I am all in for implementing this without using jquery but I do not see why this is necessary at all as a div is normally a block anyhow and if it is overwritten somewhere you could probably just change it in the stylesheet

Reply:
I think I misunderstood your question.

I took the code for the search box from the current search box implementation and adapted it for mobile. There is a [jquery line $('.inline-search').show(0);](https://github.com/python/python-docs-theme/blob/628a66466589ba42037d4aad3d676cb7d81fb6e6/python_docs_theme/layout.html#L23) which I converted to vanilla JS, both in the mobile part and the full version. I thought you were asking why I converted it to JS the way I did.

Are you, instead, asking why that line is there at all? If so, I don't know the answer :)

Reply:
Yes that is what I was referring to. I have not tried this branch yet so I do not know if removing it makes a difference though what confuses me is that the style in the stylesheet make the element an `inline-block` by default. Maybe there is a special reason for it... :shrug:

Reply:
I removed these lines, and didn't notice any change...